fxmanifest.lua

```diff
@@ -46,8 +40,7 @@ files {
 
 dependencies {
 	'oxmysql',
-	'progressbar',
-	'connectqueue'
+	'progressbar'
```

Comment:
```suggestion

```
Can we remove this from the fxmanifest?

Reply:
Not yet as the progressbar function is still needed as a dependency, people are going to get errors when they use it without that so when it's a dependency it will show they need it aswell before they can start the core